### ISSUE TITLE
/summarize 

### ISSUE THREAD

**Issue by clemsos** [QiHardware/scale-telegram-bot#34]
should be doable with some existing  keras training sets

**Comment by rejon** [comment]
Yes! Higher level love it, just send links to the bot to get a filtered view which we are already doing anyway, right?

Jon
